include/arkode/arkode_mristep.h

```diff
@@ -52,20 +52,35 @@ typedef enum
   ARKODE_IMEX_MRI_GARK3a,
   ARKODE_IMEX_MRI_GARK3b,
   ARKODE_IMEX_MRI_GARK4,
-  ARKODE_MAX_MRI_NUM = ARKODE_IMEX_MRI_GARK4
+  ARKODE_MRI_GARK_FORWARD_EULER,
+  ARKODE_MRI_GARK_RALSTON2,
+  ARKODE_MRI_GARK_ERK22a,
+  ARKODE_MRI_GARK_ERK22b,
+  ARKODE_MRI_GARK_RALSTON3,
+  ARKODE_MRI_GARK_BACKWARD_EULER,
+  ARKODE_MRI_GARK_IMPLICIT_MIDPOINT,
+  ARKODE_IMEX_MRI_GARK_EULER,
+  ARKODE_IMEX_MRI_GARK_TRAPEZOIDAL,
+  ARKODE_IMEX_MRI_GARK_MIDPOINT,
+  ARKODE_MAX_MRI_NUM = ARKODE_IMEX_MRI_GARK_MIDPOINT,
+
+  /* Default MRI coupling tables for each order */
+  MRISTEP_DEFAULT_EXPL_1 = ARKODE_MRI_GARK_FORWARD_EULER,
+  MRISTEP_DEFAULT_EXPL_2 = ARKODE_MRI_GARK_ERK22b,
+  MRISTEP_DEFAULT_EXPL_3 = ARKODE_MIS_KW3,
+  MRISTEP_DEFAULT_EXPL_4 = ARKODE_MRI_GARK_ERK45a,
+
+  MRISTEP_DEFAULT_IMPL_SD_1 = ARKODE_MRI_GARK_BACKWARD_EULER,
+  MRISTEP_DEFAULT_IMPL_SD_2 = ARKODE_MRI_GARK_IRK21a,
+  MRISTEP_DEFAULT_IMPL_SD_3 = ARKODE_MRI_GARK_ESDIRK34a,
+  MRISTEP_DEFAULT_IMPL_SD_4 = ARKODE_MRI_GARK_ESDIRK46a,
+
+  MRISTEP_DEFAULT_IMEX_SD_1 = ARKODE_IMEX_MRI_GARK_EULER,
+  MRISTEP_DEFAULT_IMEX_SD_2 = ARKODE_IMEX_MRI_GARK_TRAPEZOIDAL,
+  MRISTEP_DEFAULT_IMEX_SD_3 = ARKODE_IMEX_MRI_GARK3b,
+  MRISTEP_DEFAULT_IMEX_SD_4 = ARKODE_IMEX_MRI_GARK4
 } ARKODE_MRITableID;
 
-/* Default MRI coupling tables for each order */
-
-static const int MRISTEP_DEFAULT_3         = ARKODE_MIS_KW3;
-static const int MRISTEP_DEFAULT_EXPL_3    = ARKODE_MIS_KW3;
-static const int MRISTEP_DEFAULT_EXPL_4    = ARKODE_MRI_GARK_ERK45a;
-static const int MRISTEP_DEFAULT_IMPL_SD_2 = ARKODE_MRI_GARK_IRK21a;
-static const int MRISTEP_DEFAULT_IMPL_SD_3 = ARKODE_MRI_GARK_ESDIRK34a;
-static const int MRISTEP_DEFAULT_IMPL_SD_4 = ARKODE_MRI_GARK_ESDIRK46a;
-static const int MRISTEP_DEFAULT_IMEX_SD_3 = ARKODE_IMEX_MRI_GARK3b;
-static const int MRISTEP_DEFAULT_IMEX_SD_4 = ARKODE_IMEX_MRI_GARK4;
```

Comment:
This is inconsistent with how the defaults are defined in ERKStep and ARKStep. If we're going to define the defaults in the enum we should make it consistent across the steppers.

Reply:
Fixed. I think this would be good to do across SUNDIALS in another PR. Or at least change `int` to the enum type.